app/assets/stylesheets/application.css

```diff
@@ -66,4 +65,61 @@ time {
 
 #toast.error {
   background-color: #d44;
+}
+
+
+
+
+
+.homepage {
+  display: flex;
+  flex-direction: column;
+}
+.homepage > * > * {
```

Comment:
Can we have a slightly less intense CSS query here?

Reply:
Fixed